src/agent.cpp

```diff
@@ -162,6 +165,14 @@ int Agent::getUnum() {
   return world().self().unum();
 }
 
+void Agent::setLastActionStatusCollision(bool may_fix, bool likely_success) {
```

Comment:
Simply returning an action status is fine for some of them, but I believe I already integrated checking for may_fix for all of the behaviors defined in hfo-py, as opposed to being called directly from librcsc. Is it possible that people are using librcsc from github without hfo-py, including using the return codes? If not, then modifying librcsc will work.